esp/esp/program/modules/handlers/creditcardmodule_cybersource.py

```diff
@@ -71,13 +71,13 @@ def studentDesc(self):
         return {'creditcard': """Students who have filled out the credit card form."""}
 
     @main_call
-    @usercheck_usetl
+    @needs_student
     @meets_deadline('/Payment')
     def startpay_cybersource(self, request, tl, one, two, module, extra, prog):
         return render_to_response(self.baseDir() + 'cardstart.html', request, {})
 
     @aux_call
-    @usercheck_usetl
+    @needs_student
```

Comment:
You may want to not change these now to avoid conflicting with #1949 if not necessary.  @btidor?